src/validator.ts

```diff
@@ -7,7 +7,7 @@ import Ajv from 'ajv';
  * JSON schema validator class.
  */
 export class Validator {
-  _ajv = new Ajv();
+  _ajv = new Ajv({coerceTypes: 'array'});
```

Comment:
I don't think you are testing a case that requires array.

Reply:
you are right about that, I didn't want to add too many things to the PR, but in the other hand I like the 'arrray' coercion because it solves the case when you are expecting an array and get a direct single value instead.
I will prepare a test and let you know

Reply:
Thinking some more about it, i'm against array type coercion.

Reply:
Actually i'm not sure.